modules/mobile/docs/examples/appointments/cc_appointment.yml

```diff
@@ -39,5 +39,11 @@ data:
       patient_email: null
       best_time_to_call: null
       friendly_location_name: null
+      appointmentIEN: '123456'
+      locationId: '938'
+      eCheckinAllowed: false
 meta:
   errors: null
+  minutesCheckinEarlyWindow: 45
+  minutesCheckinLateWindow: 15
```

Comment:
changed this to match `minutesDuration` field format above listing the type of value first. also updated to integer.

Reply:
I would change this to something like:
```
checkinWindow: {
               before:
               after:
}
```

Just a little more intuitive that way to me. The descriptions of the fields can call out that it's in minutes or you can do something like `minutesBefore` if you thinks that's better. I don't feel strongly about that part.

Reply:
Oh yeah, that's better. updated

Reply:
Doesn't seem like it got pushed. Could be my github failing to sync correctly.

Reply:
weird. push failed locally on my side. should show now